apps/tdf/__tests__/pages/events/events.test.tsx

```
@@ -7,26 +7,104 @@ import { rest } from 'msw';
+    platformName: 'Test Platform',
+    PERMISSIONS: {
+      event: {
+        create: 'admin',
+      },
+    },
+  }),
+  useAuth: () => ({
+    user: null,
+  }),
+  // Mock all other exports to avoid circular dependencies
+  ConfigProvider: ({ children }: { children: React.ReactNode }) => (
```

ESLint

[error] 'React' is not defined.

(no-undef)